src/integrationTest/resources/environment.properties

```diff
@@ -8,6 +8,8 @@ document_management.url = http://document-management-api
 core_case_data.api.url=http://core-case-data-api
 send-letter.url=http://send-letter
 
+feature_toggles.async_event_operations_enabled=false
```

Comment:
I think we should run tests with this enabled as we have in production.

Reply:
reverted